server/pkg/builtin/main.go

```diff
@@ -34,6 +35,13 @@ var (
 	PropertySchemaIDStoryBlock           = id.MustPropertySchemaID("reearth/storyBlock")
 )
 
+// Overrides the plugin manifest
+// Ideally, we’d like to avoid defining this function here,
+// but it was implemented as a necessary workaround for testing pluginManifest　 on internal pkg.
+func OverridePluginManifest(pluginTestManifestJSON []byte) {
+	pluginManifest = manifest.MustParseSystemFromBytes(pluginTestManifestJSON, nil, pluginTranslationList.TranslatedRef())
+}
+
```

Comment:
As noted above, we’d prefer not to define this here,
but it's necessary for testing pluginManifest.
That said, `OverridePluginManifest` is generic and does not depend on any specific test case.